packages/transaction-controller/src/TransactionControllerIntegration.test.ts

```diff
@@ -798,12 +854,16 @@ describe('TransactionController Integration', () => {
           ],
         });
 
-        const { approvalController, networkController, transactionController } =
-          await setupController({
-            isMultichainEnabled: true,
-            getPermittedAccounts: async () => [ACCOUNT_MOCK],
-            getSelectedAddress: () => ACCOUNT_MOCK,
-          });
+        const {
+          approvalController,
+          networkController,
+          transactionController,
+          mockSelectedAccountCall,
+        } = await setupController({
+          isMultichainEnabled: true,
+          getPermittedAccounts: async () => [ACCOUNT_MOCK],
+        });
+        mockSelectedAccountCall.mockReturnValue(INTERNAL_ACCOUNT_MOCK);
```

Comment:
Let's use the same pattern here, adds a `selectedAccount` to the `setupController`, WDYT?